app/indexers/describable_indexer.rb

```diff
@@ -109,17 +107,39 @@ def archived_website?
     genres.include?('archived website')
   end
 
+  # TODO: part of https://github.com/sul-dlss/dor_indexing_app/issues/567
+  # From Arcadia, it should be:
+  #  typeOfResource is "text" and any of: issuance is "continuing", issuance is "serial", frequency has a value
+  def periodical?
+    event_selector('publication')&.note&.any? { |note| note.type == 'issuance' && note.value == 'serial' }
+  end
+
+  def pub_date
+    pub_date = PubDateBuilder.build(event_selector('publication'), 'publication')
+    return pub_date if pub_date.present?
+
+    pub_date = PubDateBuilder.build(event_selector('creation'), 'creation')
+    return pub_date if pub_date.present?
+  end
+
+  def sw_format_for_text
```

Comment:
Is there a reason this method got moved from it's previous position?

Reply:
I moved what was a short "publication_event" method to the bottom since it became a long event_selector method.